### DISCUSSION TITLE
Fantastic job!

### DISCUSSION THREAD

**Discussion by asigalov61** [cifkao/groove2groove#4] [General]
Hey Ondřej,

I hope you are doing well in these difficult times.

Fantastic job on the grove2grove! Wow! I am happy to know that your work is appreciated and grant sponsored. That's very good.

I just wanted to say hi and also share with you my attempt at style transfer. I tried algo music augmentation:
https://github.com/asigalov61/Meddleying-MAESTRO/blob/main/Meddleying_MAESTRO.ipynb
This is specifically for style transfer so check it out. Maybe you will find it helpful or useful somehow.

And I also had good results with simple RNNs if trained on compositions with a distinct specific style (think Bach/Mozart) + I also tried my own compositions, which worked pretty well. 
https://github.com/asigalov61/SuperPiano/blob/master/Super_Piano_2.ipynb

Of'course, my implementations are a far cry from yours because I did it all by myself w/o any help. 

So if you want to talk/exchange ideas/knowledge and brainstorm ideas, please let me know. I would be happy to chat and help however I can.

I prefer email (asigalov61@hotmail.com) or you can add GitHub discussion board to your project here and we talk there.

Thank you again. I will let you know my experience with your software soon as I wanted to try it. The demo was fantastic and I really enjoyed it but I want to try actual code/models on my setup/with my MIDIs.

Most sincerely,

Alex

**Comment by cifkao** [comment]
Hey Alex,

I'm glad that you appreciate my work! Also thanks for the tip about the discussion board, I didn't know about this feature!

I listened to a few of your examples on SoundCloud and they sound interesting, but without any explanation, it's hard to tell what your system is actually doing. Maybe you can write a blog post or a paper. :slightly_smiling_face: 

Of course if you have any questions, comments or ideas, feel free to [write me an e-mail](mailto:ondrej.cifka@telecom-paris.fr).

**Comment by asigalov61** [comment]
Hey Ondrej,

I apologize for the delayed response. I had to take care of some stuff IRL.

You are very welcome. I really enjoyed your work. I will explore it more once I get to it and I will definitely let you know what I think if you like.

Thank you for appreciating my music and my work as well. It means a lot to me. I totally understand that w/o the full description/explanation it is hard to understand/have an opinion about something like this and I can assure you that when I get a chance, I will do just what you have suggested and I will write a proper paper/explanation.

Basically, I am exploring new/SOTA approaches to music generation (specifically and obviously - computer music) as I find this field very exciting and unexplored. Music algos/AI is the new development in both music and computers and it will revolutionize the way we make music IMHO.

I would be more than happy to answer any questions about my software if you want so please do not hesitate to ask. I am mostly into AI/augmentation. I do not do a lot of algos. However, I just did Viterbi algo generator and also I have started working on a proper melody augmentation/chord-generator for one of my music AI implementations as I want to create a full and SOTA music AI generation system/model.

This is my work in a nutshell. Please feel free to check out my repos. Particularly check out ones on the main page/pinned ones as they are the best and most popular.

Otherwise, I hope to hear from you again.

Most sincerely from always sunny Los Angeles,

Alex


________________________________
